docs/development/build-instructions-linux.md

```diff
@@ -25,7 +25,7 @@ Follow the guidelines below for building Electron on Linux.
   Doing so permits installing Node on your own home directory as a standard user.
   Or try repositories such as [NodeSource](https://nodesource.com/blog/nodejs-v012-iojs-and-the-nodesource-linux-repositories).
 * [clang](https://clang.llvm.org/get_started.html) 3.4 or later.
-* Development headers of GTK+ and libnotify.
+* Development headers of GTK 3 and libnotify.
```

Comment:
I think the official name of GTK is `GTK+`?

Reply:
It was, but the `+` was removed [about a year ago](https://mail.gnome.org/archives/gtk-devel-list/2019-February/msg00000.html)